packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -47,6 +49,32 @@ interface TokensListProps {
 }
 
 const getTokensByTheme = (theme: ThemeVariants, props: TokensListProps): TokenCategory[] => {
+  const font = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'fonts');
+  if (font) {
+    font.info = (
+      <Callout variant="secondary">
+        <CalloutTitle as="h4">Heads up about fonts in Paste!</CalloutTitle>
+        <CalloutText>
+          Due to font-family licensing, we have no ability to redistribute certain themed fonts. Paste makes the
+          assumption that the Whitney fonts are loaded if you are using the Console theme, and Colfax for Sendgrid.
+        </CalloutText>
+      </Callout>
+    );
+  }
+
+  const fontSize = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'font-sizes');
+  if (fontSize) {
+    fontSize.info = (
+      <Callout variant="secondary">
+        <CalloutTitle as="h4">Heads up about font sizes in Paste!</CalloutTitle>
+        <CalloutText>
+          Fonts are styled using REM (Relative EM{"'"}s). They are relative to the root {'<html>'} element font size. We
```

Comment:
Hmm, this doesn't quite explain what a user is expected to do... Maybe something like:

> Font-sizes in our Paste tokens use `rem` values.  Unlike `em` values which are relative to their parent element, `rem` values are relative to the `html` element.  If you aren't using the [Theme.Provider](https://paste.twilio.design/tokens/theme-package#theme-provider) component, you must set `font-size: 100%;` on your page's `html` tag for the font-sizes to be sized correctly as 1rem=16px.

cc @SiTaggart

Reply:
this is good, I like this

Reply:
(unrelated) Is the point of using REM's so people with custom stylesheets (someone that may be visually impaired) still gets the same scaling as other people? Or is there another reason to use REM's

Reply:
We use REMs because they let give the page the flexibility to respond to browser font settings.  

Browser set to 16px font-size in Firefox(default)
![image](https://user-images.githubusercontent.com/1592327/72647261-bd4dc400-393d-11ea-8d1f-dbb351a5f3a2.png)
![image](https://user-images.githubusercontent.com/1592327/72647253-b626b600-393d-11ea-9dfc-fd1d9a54beda.png)

Browser set to 30px font-size
![image](https://user-images.githubusercontent.com/1592327/72647223-a7400380-393d-11ea-8239-1b5f19293f70.png)


We could even apply some JS to modify the `font-size` on the `html` tag to dynamically update the size only for our products, in real-time, without changing the browser setting.